iceoryx2-bb/log/src/lib.rs

```diff
@@ -161,7 +161,7 @@ static DEFAULT_LOGGER: logger::log::Logger = logger::log::Logger::new();
 #[cfg(not(any(feature = "logger_log", feature = "logger_tracing")))]
 static DEFAULT_LOGGER: logger::console::Logger = logger::console::Logger::new();
 
-const DEFAULT_LOG_LEVEL: u8 = LogLevel::Trace as u8;
+const DEFAULT_LOG_LEVEL: u8 = LogLevel::Warn as u8;
```

Comment:
What do you think about setting in to `Info`? I think is it not used in the libraries but could be helpful for examples or the iox2-cli.

Additionally, shall we create an issue to add a `-l, --log-level` option to all the examples so that it is easy to get a more detailed output if something goes wrong?

Reply:
Created the issue #396